pkg/asset/cluster/tfvars.go

```diff
@@ -877,6 +877,16 @@ func (t *TerraformVariables) Generate(parents asset.Parents) error {
 			vpcZone = fmt.Sprintf("%s-%d", vpcRegion, rand.Intn(2)+1) //nolint:gosec // we don't need a crypto secure number
 		}
 
+		if installConfig.Config.ControlPlane.Platform.PowerVS == nil || installConfig.Config.ControlPlane.Platform.PowerVS.SysType == "" {
```

Comment:
Have you tried an install-config where you don't have a `controlPlane` stanza? I think this part would panic in that case. Let's
```suggestion
		if installConfig.Config.ControlPlane == nil || installConfig.Config.ControlPlane.Platform.PowerVS == nil || installConfig.Config.ControlPlane.Platform.PowerVS.SysType == "" {
```

Reply:
Sounds like a good safe coding practice, though I have tested and seen the `ControlPlane` always populated, even with no `controlPlane:` stanza in `install-config.yaml`, at this time. Will add this check, won't hurt.

Reply:
Addressed in latest force-push. PTAL.